cluster/saltbase/salt/kubelet/default

```diff
@@ -50,8 +50,8 @@
 {% set config = "--config=/etc/kubernetes/manifests" -%}
 
 {% set manifest_url = "" -%}
-{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
-  {% set manifest_url = "--manifest-url=http://metadata.google.internal/computeMetadata/v1/instance/attributes/google-container-manifest --manifest-url-header=Metadata-Flavor:Google" -%}
```

Comment:
The `enable_manifest_url` parameter looks redundant - since you can tell whether to set the --manifest-url and --manifest-url-header flags based on whether the `manifest_url` and `manifest_url_header` parameters are set, respectively. Also, this could be error prone if it someone sets enable_manifest_url but not the other parameters, since it will cause empty flags to be passed to the apiserver. Is there a reason I'm not seeing for keeping it around?